js/plates-view/render-cross-section.js

```diff
@@ -122,6 +147,20 @@ function renderChunk (ctx, chunkData) {
   }
 }
 
+function renderChunkEarthquakes (ctx, chunkData) {
+  for (let i = 0; i < chunkData.length - 1; i += 1) {
+    const x1 = chunkData[i].dist
+    const f1 = chunkData[i].field
+
+    // Until we implement depth correctly for earthquakes, place them on the boundary beetween crust and lithosphere
+    const c1 = new THREE.Vector2(x1, f1.elevation - f1.crustThickness)
```

Comment:
I wouldn't try to tweak it that way. I'd use a correct approach (f1.elevation), wouldn't worry about imperfect depth in this PR (as that's not part of it), and then tweak the model.